packages/fonts/README.md

```diff
@@ -0,0 +1,61 @@
+# Fonts
+
+This package contains the fonts for Autodesk HIG.
+
+## Getting started
+
+### Install the package
+
+```bash
+yarn add @hig/fonts
+```
+
+## Basic usage
+
+### Import stylesheets
+
+```js
+/* ArtifaktElement font w/ font weight 100 */
+import "@hig/fonts/build/ArtifaktElement100.css";
```

Comment:
How about we publish the three weights we use in a single file together and change this example to simply say:

```
/* ArtifaktElement font w/ weights used in the design system (400, 600, and 800)  */
 import "@hig/fonts/build/ArtifaktElement.css";
```

Leave the individual weight files and all weight files for those that may want them.